blackbox/docs/general/builtins/window-functions.rst

```diff
@@ -135,20 +165,37 @@ Example::
    Window definitions order or partitioned by an array column type are
    currently not supported.
 
-.. _over:
+In the ``UNBOUNDED FOLLOWING`` case the ``window`` for each row starts with
+each row and ends with the last row in the current ``partition``. If the
+``current row`` has ``peers`` the ``window`` will include (or start with) all
+the ``current row`` peers and end at the upper bound of the ``partition``.
 
-OVER
-----
-
-Synopsis
-........
-
-::
+Example::
 
-   OVER (
-      [ PARTITION BY expression [, ...] ]
-      [ ORDER BY expression [ ASC | DESC ] [ NULLS { FIRST | LAST } ] [, ...] ]
-   )
+   cr> select dept_id, sex, count(*) OVER(PARTITION BY dept_id ORDER BY sex RANGE BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING) from employees order by 1,2,3;
```

Comment:
maybe use an alias for the `count(*)...` so that the column in the output is shorter and the table is easier to read in raw rst format.